hr_payroll_expense/README.rst

```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Payroll Expenses Reinbursements
+===============================
+
+This module allows you to add commission and reimbursement to payslip
+
+**Features list :**
+    * Add expenses reimbursement to Payslip.
+
+**For further information:**
+    * Expenses reimbursements management: http://open-net.ch/blog/la-comptabilite-salariale-suisse-avec-odoo-1/post/salaires-avec-odoo-commissions-et-notes-de-frais-78
+
+
+Installation
+============
+
+Nothing special to install this module. Just click on install in module list.
+
+Configuration
+=============
+
+No configuration is required to use this module.
+
+Usage
+=====
+
+    * To add reimbursement to the employee's payslip, you need to add the REIMB rule to the corresponding contract.
+
+    .. image:: http://s11.postimg.org/99esrh5yr/Screen_Shot_02_12_16_at_03_07_PM.png
+
+    * Then you just need to compute your payslip to find REIMB into the list.
+
+    .. image:: http://s11.postimg.org/4mhxh6xdf/Screen_Shot_02_12_16_at_03_09_PM.png
+
+
+Known issues / Roadmap
+======================
+
+V1.0.0: 2014-11-07/dco
+    * Module functions splitted from l10n_ch_hr_payroll.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/{project_repo}/issues>`_. In case of trouble, please

```

Comment:
Could you change project_repo variable to hr...and add Try me on Runbot paragraph..